### PR TITLE
Branch home 5

### PR DESCRIPTION
**Pull request by AntonWarkentin**

### CODE REVIEW

File: Home_5/Program.cs [hunk 1]
```diff
@@ -1,10 +1,38 @@
...
+﻿using System.Text.RegularExpressions;
+using System;
+
+
+namespace Home_5
 {
     internal class Program
     {
+        
+        /// <summary>
```
Comment (AliaksandrNestsiarenka) [inline Home_5/Program.cs:10]: It is better to use documentation above the class name

File: Home_5/School.cs [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+
+            for (int i = 0; i < groupsAllTogether.Length; i++)
+            {
+                groupsAllTogether[i] = new StudentsGroup();
+            }
+
+            SchoolGroups = groupsAllTogether;
+        }
+        
+        public void bestMarksInEachGroup()
```
Comment (AliaksandrNestsiarenka) [inline Home_5/School.cs]: It is better to call a method BestMarksInEachGroup with capital B

File: Home_5/School.cs [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+                RewardProvision(bestMathMarkStudent);
+                RewardProvision(bestPhysicalEducationMark);
+                RewardProvision(bestBiologyMarkStudent);
+                Console.WriteLine();
+            }
+        }
+
+        public void averageMarksInEachGroup()
+        {
+            foreach (StudentsGroup studentsGroup in this.SchoolGroups)
```
Comment (AliaksandrNestsiarenka) [inline Home_5/School.cs]: You can use just SchoolGroups, instead of this SchoolGroups

**Review by AliaksandrNestsiarenka — APPROVED** [review]

File: Home_5/School.cs (hunk 2 above)
Comment (AntonWarkentin) [reply Home_5/School.cs]: fixed

File: Home_5/School.cs (hunk 3 above)
Comment (AntonWarkentin) [reply Home_5/School.cs]: fixed

**Pull request merged**